frontend_tests/puppeteer_lib/common.ts

```diff
@@ -104,6 +104,13 @@ class CommonUtils {
         return await page.evaluate("location.href");
     }
 
+    // This function will clear the existing value of the element and
+    // replace it with the text.
+    async clear_and_type(page: Page, selector: string, text: string): Promise<void> {
+        await page.click(selector, {clickCount: 3});
```

Comment:
What does `clickCount: 3` do?  Maybe worth adding a comment above this if the idea is it clicks 3 times to select any text currently in the text box.